docs/backups.md

```diff
@@ -0,0 +1,23 @@
+# OSBS Backups
+
+## Creating Backups
+
+Creating OSBS backups is very simple. To backup all relevant data, one needs to:
+
+1. Pause OSBS by `osbs pause-builds`
+2. Wait for all running builds to finish
+3. Backup data using following commands.
+  * `oc export -o=json BuildConfig > buildconfigs.json`
```

Comment:
I'll probably use `--output=json` to make the commands explicit, I guess that's best for documentation.